x/gamm/pool-models/stableswap/pool_test.go

```diff
@@ -163,30 +185,33 @@ func TestScaledSortedPoolReserves(t *testing.T) {
 			poolAssets:     fiveUnevenStablePoolAssets,
 			scalingFactors: []uint64{1, 1, 1, 1, 1},
 			expReserves: []osmomath.BigDec{
-				baseEvenAmt.MulInt64(3),  // {"asset/c", baseEvenAmt.MulInt64(3)},
-				baseEvenAmt.MulInt64(4),  // {"asset/d", baseEvenAmt.MulInt64(4)},
-				baseEvenAmt,              // {"asset/a", baseEvenAmt},
-				baseEvenAmt.MulInt64(2),  // {"asset/b", baseEvenAmt.MulInt64(2)},
-				baseEvenAmt.MulInt64(5)}, // {"asset/e", baseEvenAmt.MulInt64(5)}},
+				baseEvenAmt.MulInt64(3),
+				baseEvenAmt.MulInt64(4),
+				baseEvenAmt,
+				baseEvenAmt.MulInt64(2),
+				baseEvenAmt.MulInt64(5),
+			},
 		},
 		"five asset pool, scaling factors = 1,2,3,4,5": {
 			denoms:         [2]string{"asset/a", "asset/e"},
 			poolAssets:     fiveUnevenStablePoolAssets,
 			scalingFactors: []uint64{1, 2, 3, 4, 5},
 			expReserves: []osmomath.BigDec{
-				baseEvenAmt,  // {"asset/a", baseEvenAmt},
-				baseEvenAmt,  // {"asset/e", baseEvenAmt},
-				baseEvenAmt,  // {"asset/b", baseEvenAmt},
-				baseEvenAmt,  // {"asset/c", baseEvenAmt},
-				baseEvenAmt}, // {"asset/d", baseEvenAmt}},
```

Comment:
Note: I removed these comments since `make format` was formatting them in an odd way (shuffling them, putting last one on newline etc.)